### PR TITLE
Fixes megafauna being immune to severity 1 explosions

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
My numbers might be too low I guess we can all argue about them

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm [hunk 1]
```diff
@@ -50,4 +50,17 @@
...
-		L.gib()
\ No newline at end of file
+		L.gib()
+
+
+/mob/living/simple_animal/ex_act(severity, target)
+	switch (severity)
+		if (1)
+			adjustBruteLoss(200)
+			return
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm]: Weird, thought I removed that

**Comment by ChangelingRain** [comment]
i dunno the ability to bomb a megafauna for 1500 damage with a bag of bombs seems pretty strong

**Comment by KazeEspada** [comment]
Its a bag of bombs. Just saying.

**Comment by Razharas** [comment]
Im pretty sure a bag of bombs should destroy anything and everything a spessmen can ever encounter

**Comment by Core0verload** [comment]
what the fuck, why do you close #18486 that does EXACTLY THE SAME THING?

**Comment by Kromgar** [comment]
His numbers are a lot lower

**Comment by Core0verload** [comment]
They are too low for bombing to be useful. Also he changes **all** simple animals, not just bosses.

**Comment by KorPhaeron** [comment]
@optimumtact 

How is this a "feature" when the strongest bombs were previously doing 0 damage to them while a welding tank would hurt them?

**Comment by ChangelingRain** [comment]
>he changes all simple animals

is it lying day that sounds like lying day

**Comment by KorPhaeron** [comment]
@ChangelingRain 

He's not lying, I made a mistake in my initial commit

**Pull request merged**